bgpd/bgp_mplsvpn_snmp.c

```diff
@@ -1411,8 +1411,8 @@ static struct bgp_path_info *bgpL3vpnRte_lookup(struct variable *v, oid name[],
 			break;
 		}
 		prefix.prefixlen = (uint8_t)name[i++];
-		*policy |= name[i++] << 8;
-		*policy |= name[i++];
+		SET_FLAG(*policy, name[i++] << 8);
```

Comment:
TBH, I don't like this pattern for encodings...

Reply:
Thanks Mr. @ton31337 for your review.
I have changed the code as per your suggestions, and will consider your inputs for future modifications aswell. thanks.